Spring-Boot-News-Service/src/main/java/com/atomshermental/springbootnewsservice/model/Comment.java

```diff
@@ -0,0 +1,16 @@
+package com.atomshermental.springbootnewsservice.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+import java.util.Date;
+@Data
+@AllArgsConstructor
+public class Comment {
+    CommentRequest commentShablon;
```

Comment:
You shouldn't use request class as field type of internal class. Instead, add the same fields as in CommentRequest class.